avaframe/com1DFA/deriveParameterSet.py

```diff
@@ -801,6 +799,94 @@ def checkRasterMeshSize(cfgSim, rasterFile, typeIndicator='DEM', onlySearch=Fals
 
     return pathToRaster
 
+def checkExtentAndCellSize(cfg, inputFile, dem, fileType):
+    """ check if extent of inputFile is within resizeThreshold of dem, if so resize and save to remeshedRasters
+
+        Parameters
+        -----------
+        cfg: configparser object
+            configuration settings
+        inputFile: pathlib Path
+            path to inputFile
+        dem: dict
+            dictionary with info on DEM
+        fileType: str
+            name of fileType
+    """
+
+    inputField = IOf.readRaster(inputFile)
+    cellSizeOld = inputField['header']['cellsize']
+    demHeader = dem['header']
+
+    if fileType == 'RELTH':
+        cT = 0.
+    else:
+        cT = float(cfg['GENERAL']['resizeThreshold'])
+
+    # compute difference of llcenter and urcenter - used for warning
```

Comment:
Mh, this to line 59 could be a separate function, as I think this might be needed more often...